src/main/java/com/petros/bibernate/util/EntityUtil.java

```diff
@@ -77,4 +114,39 @@ public static List<Object> getInsertableValues(Object entity)  {
                 })
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Retrieves the list of updatable columns for a given entity.
+     * Columns corresponding to fields annotated with {@link Id} are excluded from the list.
+     *
+     * @param entity the entity for which to retrieve the updatable columns
+     * @return the list of updatable columns
+     */
+    public static <T> List<String> getUpdatableColumns(T entity) {
```

Comment:
Looks like duplicate of getInsertableColumns. We could rid of one of them and rename the rest
 one.